src/app/components/settings/background/background.component.html

```diff
@@ -14,3 +14,5 @@ <h3 class="title">Change Background</h3>
 </div>
 <button class="trianglify btn btn-info" (click)="setTrianglify()" [disabled]="uploadingBackground">Use Randomly Generated Triangle Art for Background</button>
 <button class="btn btn-danger" (click)="deleteBackground()" [disabled]="hasDefaultBackground">Revert to Default Background</button>
+<h3 class="title" style="margin-top: 1rem">Layout</h3>
```

Comment:
Oh yeah, another thing is that you typically always want to avoid inline styling using the `style` tag. Instead, perhaps create another `.title-layout` class for the top margin, or even add a scss rule `.title:nth-of-type(2) { ... }` so that the second `.title` in the component will have the top margin automatically

Reply:
Also, I'd consider changing "Layout" to "Homepage Layout" just to make sure it's clearer, and putting it above the change background because that way it's more visible above the clutter of the background form